renovate-changesets/dependencyConfig.ts

```diff
@@ -0,0 +1,17 @@
+import * as core from '@actions/core';
+
+export type DependencyMangerConfig = {
+  branchPrefix: string;
+  changesetPrefix: string;
+};
+
+export const dependencyMangerConfig: Record<string, DependencyMangerConfig> = {
+  renovate: { branchPrefix: 'renovate/', changesetPrefix: 'renovate' },
+  dependabot: { branchPrefix: 'dependabot/', changesetPrefix: 'dependabot' },
```

Comment:
This seems overly complicated without providing any additional value. The `changesetPrefix` is identical to the `key` and the `branchPrefix` is just `${key}/`. Why not simpliffy this to an array of accepted values (instead of an object) and you can derive the rest from the dependency manager name alone?